docs/proposals/ldap-group-sync.md

```diff
@@ -0,0 +1,295 @@
+# Allowing OpenShift Groups to be Synced With Sources on an LDAP Server and Making it Extensible for Other External Sources
+
+___
+Note: in this proposal, if a reference is prefixed with "OpenShift" or is typeset in `monospace`, it refers to an OpenShift object or field. For instance, the following table enumerates what references to OpenShift resources and LDAP entries may look like:
```

Comment:
If I added OpenShift before everything in `monospace` this document would double in length. This is the best compromise but even without this disclaimer that should be a fairly obvious distinction (code = `monospace`, LDAP references = not Go objects)